web/modules/custom/dpl_library_agency/src/Form/GeneralSettingsForm.php

```diff
@@ -247,9 +252,37 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
   }
 
   /**
+   * Validates interest_periods_config and default_interest_period_config.
+   *
+   * The interest_periods_config is validated for the format [days]-[label].
+   *
+   * The validation for default_interest_period_config is checking
+   * that the selected default value is present in interest_periods_config.
+   *
    * {@inheritdoc}
+   *
+   * @throws \Safe\Exceptions\PcreException
    */
   public function validateForm(array &$form, FormStateInterface $form_state): void {
+    $interestPeriods = $form_state->getValue('interest_periods_config');
+    $defaultInterestPeriod = $form_state->getValue('default_interest_period_config');
+
+    $interestPeriodsArray = explode(PHP_EOL, $interestPeriods);
```

Comment:
According to drupal coding style variabels should use underscore.
I am not fond of using the data type as a part of the variable name. It should be implicit. So I don't think it is needed to state that it is an array in: `$interestPeriodsArray`

Reply:
Updated this. Please see latest commit.

Reply:
👍